src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx

```diff
@@ -14,6 +14,14 @@ describe('RowUserBase', () => {
     await screen.findByText(randomUser.name);
   });
 
+  it('basic render with empty Box instead of Menu if there is not actions defined', async () => {
+    const randomUser = createRandomUser();
+    renderWrappedInTestingProvider(<UserRowBase user={randomUser} />);
+    const menuBox = await screen.findByTestId('menuBox')
+    const emptyBox = menuBox.querySelector(".emptyBox")
+    expect(emptyBox).toBeInstanceOf(HTMLElement);
```

Comment:
Why not directly search for the presence of the element by also adding a data-testid to it?

```
await screen.findByTestId('emptyBox')
```

Reply:
Hi @PanchoutNathan thanks for feedback 🙏🏾.
I just pushed updates.

Reply:
you can remove ```expect(emptyBox).toBeInstanceOf(HTMLElement);```
Because the return type of  
```const emptyBox = await screen.findByTestId('emptyBox');```   
is a ```Promise<HTMLElement>``` and fails the test if it doesn't find it.